0.download_data/metadata/platemap_NF1_plate6.csv

```diff
@@ -1,4 +1,4 @@
-WellRow,WellCol,well_position,gene_name,genotype,Institution ,seed_density
+WellRow,WellCol,well_position,gene_name,genotype,Institution,seed_density

```

Comment:
nice fix! Just checking that you reprocessed these data now after updating the platemap?

Reply:
Yup! I had to do that to even process the data because I couldn't access the column properly.